packages/hydrated_bloc/README.md

```diff
@@ -171,6 +171,44 @@ HydratedBlocOverrides.runZoned(
 );
 ```
 
+## Testing 
+In order to successfully test your code - you need to use a Mock for your storage.
+For example - [Taken from Flutter Weather Example](https://github.com/felangel/bloc/blob/master/examples/flutter_weather/test/helpers/hydrated_bloc.dart)
+
+Define a Mock Storage
+
+```dart
+class MockStorage extends Mock implements Storage {}
+
+FutureOr<T> mockHydratedStorage<T>(
+  FutureOr<T> Function() body, {
+  Storage? storage,
+}) {
+  return HydratedBlocOverrides.runZoned<T>(
```

Comment:
This is outdated since we deprecated the BlocOverrides API. Are you able to update the PR to the latest syntax?